apps/autonolas-registry/components/ListAgents/mint.jsx

```diff
@@ -3,6 +3,8 @@ import { useRouter } from 'next/router';
 import { Typography } from 'antd';
 import { notifyError, notifySuccess } from '@autonolas/frontend-library';
 
+import { getEstimatedGasLimit } from 'libs/util-functions/src';
```

Comment:
Please remind me if this `getEstimatedGasLimit` function fallsback into a default gas limit value when the actual gas estimation function fails to execute?

Reply:
it does, the `500_000` is default